acceptance/config/nodes/huaweios-6-powerpc.yaml

```diff
@@ -0,0 +1,14 @@
+---
+HOSTS:
+  huaweios-6-powerpc:
+    roles:
+      - agent
+    platform: huaweios-6-powerpc
+    hypervisor: none
+    ip: 10.0.7.26
+    vmhostname: huawei-ce6850-2-debian-vm-eth0.ops.puppetlabs.net
+CONFIG:
```

Comment:
@LuvCurves since hypervisor is 'none', I presume that the pooler CONFIG options are not required. If so, this section can be removed.